frontends/web/src/components/guide/guide.tsx

```diff
@@ -61,14 +62,30 @@ interface ProvidedProps {
 
 type Props = ProvidedProps & SharedProps & TranslateProp;
 
+// guide entries are fetched from the translation files. Manual fallback to is implemented in case an
+// array element is undefined in the current language (can be null if not translated in order, for
+// example).
+function getGuideEntries(t, screen): ImplicitEntry[] {
+    const key = 'guide.' + screen;
+    // For now, fallback to the main fallback language, not cascading.
+    const entriesFallback = i18n.getFixedT(i18n.languages[i18n.languages.length - 1])(key, { defaultValue: [] });
+    const entriesCurrentLng = t(key, { defaultValue: [] });
```

Comment:
I have quite a strong preference to call such variables `entriesCurrentLanguage` (or `entriesOfCurrentLanguage` or `currentLanguageEntries`). It's mentally just so much easier to read/parse. I first thought `lng` might refer to `length`. 😬